e2e_test/src/independent/replica_test.rs

```diff
@@ -139,12 +287,326 @@ pub mod test {
         let resp = client.post(SERVER_URL, "select count(*) from ma").unwrap();
         assert_eq!(resp.status(), status_code::OK);
         let actual = resp.text().unwrap();
-        let expected = format!("{}", write_count);
+        let expected = format!("{}", 0);
         println!("\nselect count(*): {}", actual);
         println!("expected: {}", expected);
         assert!(actual.contains(&expected));
```

Comment:
Use assert_eq! to compare the result of `select count`.

Reply:
Done